meta-lmp-base/recipes-security/optee/optee-client/0001-tee-supplicant-rpmb-switch-to-MMC_IOC_MULTI_CMD.patch

```diff
@@ -0,0 +1,277 @@
+From 5da4da99e0bb3d694e93046207ae4ce3b5545043 Mon Sep 17 00:00:00 2001
+From: Ricardo Salveti <ricardo@foundries.io>
+Date: Tue, 9 Feb 2021 19:31:52 -0300
+Subject: [PATCH] tee-supplicant: rpmb: switch to MMC_IOC_MULTI_CMD
```

Comment:
Does it make sense to add also [FIO fromtree]?

Reply:
Since this is just in OE, I went with the OE-way to track patch status instead:

`Upstream-Status: Backport
`